includes/helpers/functions-global.php

```
@@ -0,0 +1,247 @@
+        }
+    }
+}
+
+/**
+ * Render a follow button
+ *
+ * @param array $args
+ * @return string
+ */
+if (!function_exists('zippicks_render_follow_button')) {
+    function zippicks_render_follow_button($args = []) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)